XcodeServerSDK/XcodeServer.swift

```diff
@@ -415,6 +447,32 @@ public extension XcodeServer {
         }
     }
     
+    /**
+    XCS API call for retrievieng all available integrations on server.
+    
+    - parameter integrations:   Optional array of integrations.
+    - parameter error:          Optional error.
+    */
+    public func getIntegrations(completion: (integrations: [Integration]?, error: NSError?) -> ()) {
```

Comment:
**Thank you**‼️

I hate JS... 😕 Can't find anything in this mess! Hope Swift will replace it soon 😆